baklava/src/constants.ts

```diff
@@ -1 +1,65 @@
+import {
+    ipcMain,
+    globalShortcut,
+    BrowserWindow
+} from "electron";
+
 export const __prod__ = process.env.NODE_ENV === "production";
+export const REQUEST_TO_SPEAK_KEY = "@keybind/invite";
+export const INVITE_KEY = "@keybind/invite";
+export const MUTE_KEY = "@keybind/mute";
+export const CHAT_KEY = "@keybind/chat";
+export const PTT_KEY = "@keybind/ptt";
+export let CURRENT_REQUEST_TO_SPEAK_KEY = "Control+8";
+export let CURRENT_INVITE_KEY = "Control+7";
```

Comment:
the bindings can change for keys

Reply:
Yes im aware im using those variables for storing the current keybinds in cache. Is that bad practice? If yes please advise me on what to do?

Reply:
The function right under these lines `RegisterKeybinds` sets listeners for any KeyBinds changing and removes the previous ones and sets the new ones as GlobalShortcuts

Reply:
nvm, I was confused about how this worked, but I see it now and it looks good